aptos-move/framework/aptos-framework/sources/aptos_account.spec.move

```diff
@@ -60,17 +125,121 @@ spec aptos_framework::aptos_account {
+        let balance_source = coin_store_source.coin.value;
+
+        requires forall i in 0..len(recipients):
+            recipients[i] != account_addr_source;
+
+        requires exists i in 0..len(recipients):
+            amounts[i] > 0;
+
+        // cointype should not be aptoscoin, otherwise it will automaticly create an account.
+        // meanwhile, aptoscoin has already been proved in normal tranfer
+        requires type_info::type_of<CoinType>() != type_info::type_of<AptosCoin>();
+
+        aborts_if len(recipients) != len(amounts);
+
+        aborts_if exists i in 0..len(recipients):
+                !account::exists_at(recipients[i]) && length_judgment(recipients[i]);
+                
+        aborts_if exists i in 0..len(recipients):
+                !account::exists_at(recipients[i]) && (recipients[i] == @vm_reserved || recipients[i] == @aptos_framework || recipients[i] == @aptos_token);
+
+        ensures forall i in 0..len(recipients):
+                (!account::exists_at(recipients[i]) ==> !length_judgment(recipients[i])) &&
+                    (!account::exists_at(recipients[i]) ==> (recipients[i] != @vm_reserved && recipients[i] != @aptos_framework && recipients[i] != @aptos_token));
+
+        //coin::withdraw properties
+        aborts_if exists i in 0..len(recipients):
+            !exists<coin::CoinStore<CoinType>>(account_addr_source);
+        aborts_if exists i in 0..len(recipients):
+            coin_store_source.frozen;
+        aborts_if exists i in 0..len(recipients):
+            global<coin::CoinStore<CoinType>>(account_addr_source).coin.value < amounts[i];
+
+        // deposit properties
+        aborts_if exists i in 0..len(recipients): 
+            exists<coin::CoinStore<CoinType>>(recipients[i]) && global<coin::CoinStore<CoinType>>(recipients[i]).frozen;
+
+        // guid properties
+        aborts_if exists i in 0..len(recipients):  
+            account::exists_at(recipients[i]) && !exists<coin::CoinStore<CoinType>>(recipients[i]) && global<account::Account>(recipients[i]).guid_creation_num + 2 >= account::MAX_GUID_CREATION_NUM;
+        aborts_if exists i in 0..len(recipients):  
+            account::exists_at(recipients[i]) && !exists<coin::CoinStore<CoinType>>(recipients[i]) && global<account::Account>(recipients[i]).guid_creation_num + 2 > MAX_U64;
+
+        // register_coin properties
+        aborts_if exists i in 0..len(recipients):  
+            !coin::is_account_registered<CoinType>(recipients[i]) && !type_info::spec_is_struct<CoinType>();
+        aborts_if exists i in 0..len(recipients):  
+            !coin::is_account_registered<CoinType>(recipients[i]) && !can_receive_direct_coin_transfers(recipients[i]);
+
     }
 
     spec deposit_coins<CoinType>(to: address, coins: Coin<CoinType>) {
-        // TODO: missing aborts_if spec
-        pragma verify=false;
+        use aptos_std::type_info;
+        pragma verify=true;
```

Comment:
Remove this line.

Reply:
Should I remove use aptos_std::type_info? it seemed affected the result